src/actions/incidentAction.jsx

```diff
@@ -1,23 +1,29 @@
+import * as axios from 'axios';
 import { FETCH_INCIDENTS_SUCCESS } from './actionTypes';
 
 const url = () => {
-    return 'www.url.com';
+    // get all incidents
+    return 'https://private-3b686-wire3.apiary-mock.com/incidents';
 };
 
-const apiKey = 'apiKey';
+// const apiKey = 'apiKey';
```

Comment:
You can add a comment here if this is to be used sometime soon/later.

Reply:
I have removed it because it will be in another file so that it can be accessed by all code requiring it. 👍